.github/workflows/appimage/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:bionic
+
+RUN apt-get update -q -y && \
+    apt-get install -q -y \
+        python3-pip python3-setuptools patchelf desktop-file-utils appstream \
+        curl git \
+        build-essential cmake extra-cmake-modules ninja-build \
+        libarchive-dev libsqlite3-dev \
+        libappindicator-dev libxcb-keysyms1-dev \
+        qtbase5-dev libqt5webkit5-dev libqt5x11extras5-dev \
```

Comment:
Please install: `zsync` it's required to generate the AppImage update information

Reply:
In my local testing, I installed `zsync` and added `update-information` from https://github.com/zealdocs/zeal/pull/1200#issuecomment-616183454, but I don't see `.zsync` file being generated.